src/main/java/competition/subsystems/pose/PoseSubsystem.java

```diff
@@ -79,6 +81,8 @@ public PoseSubsystem(XGyroFactory gyroFactory, PropertyFactory propManager, Driv
                this.setCurrentPoseInMeters(getVisionAssistedPositionInMeters());
            }
         });
+        // creating matchtime doubleProperty
+        matchTime = propManager.createEphemeralProperty("Time", DriverStation.getMatchTime());
```

Comment:
You will need to update the value of this property in the periodic() method of the subsystem.

matchTime.set(DriverStation.getMatchTime());

Reply:
```
    @Override
    public void periodic() {
        super.periodic();
    }
```